README.md

```diff
@@ -3,9 +3,10 @@
 # [ghuser.io](https://github.com/ghuser-io/ghuser.io)'s database
 
 This repository:
-* Serves as database for the [ghuser.io](https://github.com/ghuser-io/ghuser.io) Reframe app.\
-The DB consists of the [JSON files in `data`](data/).
-* Provides scripts to update the database. 
+
```

Comment:
I prefer without but I don't mind :)

Reply:
Some markdown renderers (I have the Bitbucket one especially in mind) require empty lines around lists and codeblocks, among others.